dolphinscheduler-task-plugin/dolphinscheduler-task-spark/src/main/java/org/apache/dolphinscheduler/plugin/task/spark/SparkTask.java

```diff
@@ -195,9 +198,29 @@ private List<String> populateSparkOptions() {
         }
 
         // bin/spark-sql -f fileName
+        String sqlContent = "";
+        String resourceFileName = "";
```

Comment:
We should put these variable declarations inside the following if-block.

Reply:
Sure